libs/api-client/src/api-client.ts

```diff
@@ -31,26 +38,26 @@ interface UserTokens {
   refreshToken: string
 }
 
-interface LibConfig {
-  debug?: boolean
-  baseUrl?: string
-  dataset?: string
-}
-
 interface LoginParams {
   accessToken?: string | null
   refreshToken?: string | null
 }
-
+export type ApiVersion = '' | 'v1' | 'v2'
```

Comment:
With [this change](https://github.com/GlobalFishingWatch/frontend/pull/1546/files#diff-096d4e97610c7a4951df4066d454cc29cdbc9a060252a6bd9b9ee18483f2bb99R249) and knowing know that every endpoint is now prefixed by the version.

does it makes sense to keep the `''` ?

Reply:
I am still using it in const dataset = await GFWAPI.fetch<Dataset>(`/datasets/${LATEST_CARRIER_DATASET_ID}`, { version: '' })

I dont know if it will be needed in another apps